modules/ROOT/pages/_partials/registering-a-mobile-app.adoc

```diff
@@ -1,6 +1,6 @@
 include::{partialsdir}/attributes.adoc[]
 
-= Registering a Mobile App
+= Registering a {mobile-client}
```

Comment:
you register a mobile app (to create a mobile-client)
ie there's no mobile-client before you register, and the only thing that exists (even if it's just in your head) is a mobile app